README.md

```diff
@@ -34,6 +32,20 @@ The API is a golang HTTP server located in `src/api/`.
 
 More API docs are [here](src/api/README.md).
 
-## Status of the Project
+# Running a development environment
+
+We leverage [docker](https://www.docker.com) (via `docker-compose`) to provide a multi-tier setup for development.
+
+Running `docker-compose up` from the root directory will expose:
+
+* API backend endpoint via `http://{your-docker-machine-ip-address}:8080`
+* UI frontend via `http://{your-docker-machine-ip-address}:4200`  
+
+**IMPORTANT**: If your Docker Machine hostname is different than *localhost*, you need to change
```

Comment:
The UI assumes that the backend is under localhost. This is a problem with docker based setups that could be fixed in two ways:

1 - Use something like [Dinghy](https://github.com/codekitchen/dinghy) and enforce static hostnames for frontend and backend.
2 - Offer a way to the developer to change the backend hostname value.

Regarding #1, the problem is that it adds the requirement of running the dinghy-http-proxy in front which might be an overkill for linux and docker for mac users.

In the other hand, it seems that docker for mac uses localhost in the same way than the default Linux does.

Regarding #2, the problems is that angular-cli does not offer a way to extend the [WebPack configuration](https://github.com/angular/angular-cli/issues/1656) so we can not load this Hostname Value based on an environment variable.

So for now and while this changes, we ask the developer to change the hardcoded value in the `config.ts`.